### DISCUSSION TITLE
[sanitize-html]: Missing `parseStyleAttributes` in the sanitize function options

### DISCUSSION THREAD

**Discussion by vishal-tyagi-github** [DefinitelyTyped/DefinitelyTyped#64725] [Issues with a @types package] [answered]
Hello, 

the `sanitize.IOptions` interface missing the property `parseStyleAttributes` as described here https://github.com/apostrophecms/sanitize-html#ignore-style-attribute-contents

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1d880d4a5b97a908d67e3aedf02cad9adba51c1e/types/sanitize-html/index.d.ts#L55

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
fix in #64726

**Reply by peterblazejewicz** [reply]
updated
